packages/client/src/arpa_reporter/scss/index.scss

```diff
@@ -86,7 +86,7 @@ hr {
   color: $positive-default;
 }
 
-.usdr-tab-nav-link {
+.usdr-link {
```

Comment:
Note to reviewer: renamed the class to be general as it only contains color setting.